git_log_dots.rst

```diff
@@ -15,7 +15,7 @@ Logging without dots
 
 Two dots is the default in ``git log``.  That is, if you do::
 
-    git log master topic
+    git log ^master topic
```

Comment:
It's true that `git log master topic` has the same meaning as `git log ^master topic` (see <https://git-scm.com/docs/git-rev-parse#_specifying_ranges>), but I think it would be confusing to add the `^` here, without explaining it, and without using it elsewhere.

Reply:
No worries, we can close the MR. I stumbled upon your page as Google not cited your page ( thanks for the information by th way!).

I made the suggestion because the next sentence  about the equality between this and the two-dot version confused me. 

The sentence gives the impression that `git log foo bar` and `git log foo..bar` are equal but this description works iff `foo=master/main`. The version with negation is true for any `foo`.